### PR TITLE
Add initial digital reproduction tooling

### PR DESCRIPTION
**Pull request by niklasl**
This  contains:

- Updated marcframe mappings and mechanisms to produce the same MARC from simplified descriptions (utilizing embellished data as source for the redundancies in MARC).
- A process for creating digital reproductions (just a whelktool script; might be obsolete if we use the REST API).
- Batch job for "digibest" using the a sample backlog.

### CODE REVIEW

**Review by niklasl — COMMENTED** [review]
Note that this is a manual mapping which mirrors (one of) the mappings defined in https://github.com/libris/definitions/pull/341

Later on, if those mappings have landed and have a more powerful mechanism of getting at term-matching relationships in core, we could switch to using that here, and remove these hard-coded patterns and term references.

File: whelk-core/src/main/resources/ext/marcframe.json [hunk 1]
```diff
@@ -2136,109 +2245,329 @@
...
                   }
                 ]
               }
             }
+          }
+        ]
+      },
+      {
+        "type": "InjectWhenMatchingOnRevert",
+        "_TODO": "This should be closeMatch/broader-driven (and computed on search)",
```
Comment (niklasl) [inline whelk-core/src/main/resources/ext/marcframe.json:2412]: Note that this is a manual mapping which mirrors (one of) the mappings defined in libris/definitions#341

Later on, if those mappings have landed and have a more powerful mechanism of getting at term-matching relationships in core, we could switch to using that here, and remove these hard-coded patterns and term references.

File: whelk-core/src/main/resources/ext/marcframe.json [hunk 2]
```diff
@@ -2136,60 +2245,127 @@
...
+        "produceMissing": {
+          "produceProperty": "marc:publicNote",
+          "showProperties": [
+            {"property": ["usageAndAccessPolicy", "label"]},
+            {
+              "property": ["publisher", "name"],
+              "useValueFormat": {
+                "contentFirst": " via ",
+                "contentAfter": " och ",
+                "contentLast": ""
```
Comment (olovy) [inline whelk-core/src/main/resources/ext/marcframe.json:2259]: :sunglasses:

**Review by olovy — APPROVED** [review]
LGTM!

**Pull request merged**